process/smartContract/process.go

```diff
@@ -251,6 +263,7 @@ func NewSmartContractProcessor(args ArgsNewSmartContractProcessor) (*scProcessor
 	log.Debug("smartContract/process: enable epoch for created async callback on cross shard only", "epoch", sc.createdCallBackCrossShardOnlyEnableEpoch)
 	log.Debug("smartContract/process: enable epoch for optimize gas used in cross mini blocks", "epoch", sc.optimizeGasUsedInCrossMiniBlocksEnableEpoch)
 	log.Debug("smartContract/process: enable epoch for return as failure when saving under protected key", "epoch", sc.saveKeyValueUnderProtectedErrorEnableEpoch)
+	log.Debug("smartContract/process: enable epoch for cleaning up created scrs which contain only information", "epoch", sc.cleanupSCRDataEnableEpoch)
```

Comment:
`smartContract/process: enable epoch for cleaning up created scrs that are informative only`

Reply:
done